src/shared/credentials/defaultCredentialSelectionDataProvider.ts

```diff
@@ -40,14 +41,39 @@ export class DefaultCredentialSelectionDataProvider implements CredentialSelecti
     private readonly _credentialsMru: CredentialsProfileMru
     private readonly helpButton = createHelpButton(credentialHelpUrl)
 
-    public constructor(public readonly existingProfileNames: string[], protected context: ExtensionContext) {
+    public constructor(public readonly existingProfileNames: string[], protected context: vscode.ExtensionContext) {
         this._credentialsMru = new CredentialsProfileMru(context)
     }
 
     public async pickCredentialProfile(
         input: MultiStepInputFlowController,
+        actions: vscode.QuickPickItem[],
         state: Partial<CredentialSelectionState>
-    ): Promise<QuickPickItem> {
+    ): Promise<vscode.QuickPickItem> {
+        // Remove this stub after we bump minimum to vscode 1.64
+        const QuickPickItemKind = (vscode as any).QuickPickItemKind
+        const menuTop: vscode.QuickPickItem[] = [
+            // vscode 1.64 supports QuickPickItemKind.Separator.
+            // https://github.com/microsoft/vscode/commit/eb416b4f9ebfda1c798aa7c8b2f4e81c6ce1984f
+            ...(!QuickPickItemKind
+                ? []
+                : [
+                      {
```

Comment:
How does C9 react to separators? I've never tested it.

Reply:
Good callout, it ignores them:

<img width="287" alt="image" src="https://user-images.githubusercontent.com/55561878/182208591-99be57af-770f-4fd6-adad-ac2458ec47e2.png">